pyproject.toml

```diff
@@ -1,9 +1,9 @@
 # Copyright 2024 Canonical Ltd.
 # See LICENSE file for licensing details.
 [project]
-name = "paas-app-charmer"
-version = "1.3.1"
-description = "Companion library for PaaS app charmer."
+name = "paas-charm"
+version = "1.0.0"
```

Comment:
I think we should keep the version the same, I believe charmcraft has a minimum version specified in the template and we should probably keep updating that to make sure people get the latest when they start

Reply:
I initially thought that way, however it is a totally new project and looked strange in Pypi.

charmcraft will have the dependency we prefer (initially I have put `paas-charm>=1.0,<2`), see https://github.com/canonical/charmcraft/pull/1961 for a possibility.

As the old paas-app-charmer depends on `paas-charm<2`, the only real requirement for the initial release is that the version is in the major 1.